util/travis_build.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# if docker is installed - call make within the qmk docker image
+if docker -v &> /dev/null; then
```

Comment:
```suggestion
if command -v docker >/dev/null; then
```
For existence checks, I suggest using the `command -v` shell builtin.

Reply:
Done